Packs/CrowdStrikeFalcon/Playbooks/playbook-Command_and_Scripting_Interpreter.yml

```diff
@@ -0,0 +1,1717 @@
+    type: start
+    task:
+      id: 406f2ce0-3535-471d-836d-1bbbae057971
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "96"
+      - "97"
+      - "98"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1410,
+          "y": -1040
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 0b781a8b-1626-4995-8b19-723b7a7a68fd
+    type: title
+    task:
+      id: 0b781a8b-1626-4995-8b19-723b7a7a68fd
+      version: -1
+      name: Analysis
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "61"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1410,
+          "y": -705
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: 893c7ef4-e2a2-48cf-83e6-0dde5035a0c7
+    type: regular
+    task:
+      id: 893c7ef4-e2a2-48cf-83e6-0dde5035a0c7
+      version: -1
+      name: Close Alert - True Positive
+      description: Closes the alert in Cortex XSIAM
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "29"
+    scriptarguments:
+      closeNotes:
+        simple: Resolved As True Positive alert - Threat Handled
+      closeReason:
+        simple: Resolved
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1220,
+          "y": 2990
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: dfb7dad8-f1dd-488d-8430-399dfab6fe01
+    type: title
+    task:
+      id: dfb7dad8-f1dd-488d-8430-399dfab6fe01
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 350,
+          "y": 3170
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: 9e661cc1-2b64-490e-86b3-21eb7f296429
+    type: condition
+    task:
+      id: 9e661cc1-2b64-490e-86b3-21eb7f296429
+      version: -1
+      name: 'Found any malicious behavior? '
```

Comment:
In this task you only checks the results of the cmd analysis script, what about the reputation if you already run it the task before?

Reply:
Legit. Adding it as an 'OR' option as well.